src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/RolesApiTest.java

```diff
@@ -144,7 +144,8 @@ public void testRolesApi() throws Exception {
 
         // hidden role
         response = rh.executeGetRequest("/_opendistro/_security/api/roles/opendistro_security_internal", new Header[0]);
-        Assert.assertEquals(HttpStatus.SC_NOT_FOUND, response.getStatusCode());
+        Assert.assertTrue(response.getBody().contains("\"hidden\":true"));
+        Assert.assertEquals(HttpStatus.SC_OK, response.getStatusCode());
```

Comment:
nit: it is better to check for the status code first. When test does not fail, it does not matter, but may help with debugging if test fails. The same applies to all other places where response returns success code.

Reply:
done